recipes/hexdump/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
```

```suggestion
conan_basic_setup(TARGETS)

find_package(fernandovelcic-hexdump CONFIG REQUIRED)
```

Let's try targets, because ${CONAN_LIBS} has no effect, as it's a header-only project.